tools/query_breakdown/src/main/java/com/google/bigquery/Main.java

```diff
@@ -1,13 +1,17 @@
+package com.google.bigquery;
+
+import java.io.IOException;
+
 /**
  * this file is the main file for the command line tool. The first argument is a txt file
  * containing the queries. The second argument, which is optional, is a time limit that we
  * impose on processing a single query. The third argument, which is optional, is a file
  * that we want to write the results to.
  */
-public class QueryBreakdown {
+public class Main {
   public static void main(String[] args) {
     String inputFile = null;
-    double timeLimit = 0.0;
+    int errorLimit = 0;
     String outputFile = null;
 
     if (args[0] != null) {
```

Comment:
The logic seems wrong here. Seems it should be 
```
if (args[0] == null) {
```

Reply:
I think the logic here was that if an argument was provided, the variable would be initialized. I maintained the same logic for my new code--let me know if this is wrong!

Reply:
in this case, the code block should be something like this:

```java
if (args[0] != null) {
  inputFile = args[0];
} else {
  System.out.println("You have not specified the input file");
  return -1;
}
```

In the current form, `inputFile` is not initialized.

Reply:
Gotcha, I changed the code in main.java to reflect the CLI interface, so I do believe the current way should resolve this issue. Resolving for now!